stimela/kitchen/cab.py

```diff
@@ -313,12 +315,19 @@ def stringify_argument(name, value, schema, option=None):
             if schema.dtype == "bool":
                 if key_value:
                     args += [f"{name}={value}"]
+                elif key_value_command_line:
+                    val = get_policy(schema, "explicit_" + str(value).lower()) or value
+                    args += [f"{option}={val}"]
                 else:
                     explicit = get_policy(schema, 'explicit_true' if value else 'explicit_false')
                     args += [option, str(explicit)] if explicit is not None else ([option] if value else [])
             else:
+                
                 value = stringify_argument(name, value, schema, option=option)
                 if type(value) is list:
+                    if key_value_command_line:
+                        if len(value) > 1:
+                            value = [f"{value[0]}={value[1]}"]
```

Comment:
What if ``value`` has more than two elements? This will result in everything past the second element being ignored. Probably better to throw an error here, since for an input of [X,Y], ``--option=X Y`` is not a sensible construction anyway. More generally, ``repeat_policy=list`` can't really work with key=value syntax in any sensible way, so we should flag that up front during cab validation.

Reply:
Yes, this list can only have two elements in this case. Could raise an exception if this is not the case.